src/test/NewGame.test.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Enzyme, { mount } from "enzyme";
+import NewGame from "../components/NewGame";
+import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
+import Button from "react-bootstrap/Button";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("<NewGame />", () => {
+  it("renders <NewGame /> component", () => {
+    const wrapper = mount(<NewGame />);
+    expect(wrapper.find(NewGame)).toHaveLength(1);
+  });
+
+  it("renders <NewGame /> component that has a button", () => {
+    const wrapper = mount(<NewGame />);
+    console.log(wrapper.debug());
+    expect(wrapper.find("Button")).toHaveLength(1);
```

Comment:
Sometimes you want to have a test confirm what the button should say. You can assert that it has a specific text string